hudi-common/src/main/java/org/apache/hudi/common/util/collection/RocksDBDAO.java

```diff
@@ -378,9 +416,10 @@ public <T extends Serializable> Stream<Pair<String, T>> prefixSearch(String colu
    * Return Iterator of key-value pairs from RocksIterator.
    *
    * @param columnFamilyName Column Family Name
-   * @param <T>              Type of value stored
+   * @param <T>              Type of key stored
+   * @param <R>              Type of value stored
    */
-  public <T extends Serializable> Iterator<T> iterator(String columnFamilyName) {
+  public <T extends Serializable, R extends Serializable> Iterator<Pair<T, R>> iterator(String columnFamilyName) {
```

Comment:
Why I refactor this?
1. This is an mistake because the method comment says to return all Key-value pairs, but only value is returned. It makes more sense to return key-value pairs here.
2. If I do not return key-value pairs, I cannot filter the keys after `RocksDbDiskMap::iterator(filter)`